### PR TITLE
Uhm 5691

### PR DESCRIPTION
**Pull request by louidorjp**
@lnball @brandones 

Please review this PR.

### CODE REVIEW

**Comment by lnball** [comment]
@mogoodrich @louidorjp This is a small change but every merge has so many formatting changes -- especially style and javascript.  Can you look at the differences?  It would be easier if the formatting was consistent.  

I won't have time to test it today and not back until next Wednesday.  Mike Seaton is away.  Maybe Mark or @brandones can review.

**Comment by mogoodrich** [comment]
@louidorjp @lnball I can look at this today or tomorrow.

Louidor, not sure if your IDE is changing tabs into whitespace or the other way around, leading to all the diffs?

But, Ellen, there is a handy feature in Github for hiding whitespace changes than can make this a lot easiest to look at:

**Comment by lnball** [comment]
Thanks @mogoodrich.  "hide whitespace" is a great help.  The code looks good.  

But not sure about merging when it will lose all the formatting improvements on style and javascript.

File: configuration/pih/scripts/global/zl.js
Comment (mogoodrich) [inline configuration/pih/scripts/global/zl.js:55]: Just out of curiosity, what is at index 5 and index 448?

**Review by mogoodrich — APPROVED** [review]
I'm not too up-to-speed on this code, but looks good to me.  I had one question that I added.

Also, it looks like there's a merge conflict?

**Comment by mogoodrich** [comment]
> But not sure about merging when it will lose all the formatting improvements on style and javascript.

@lnball  do you mean that you don't want the formatting changes that were made as part of the commit?  They will still get merged in (which maybe isn't what we want).

File: configuration/pih/scripts/global/zl.js
Comment (mogoodrich) [inline configuration/pih/scripts/global/zl.js:4]: @brandones @louidorjp  looks like all this is reformating the tabs to be 4 spaces vs 2 spaces.

Do we have a sense of what the standard is for JavaScript? I don't have a strong preference, but I figure we should all do the same so we all aren't reformatting every time we commit.  Brandon, what does OpenMRS 3.0 use?

Also, Brandon suggested we add an .editorconfig file to start enforcing this automatically... do one of you want to look into setting that up?

File: configuration/pih/styles/global/zl.css
Comment (mogoodrich) [inline configuration/pih/styles/global/zl.css:2]: And the same question here... this looks to be shifting the css spacing from 6 spaces to 4 spaces (I think?).  Let's standardize... but defer to @louidorjp  and @brandones  on what they want to use.

File: configuration/pih/htmlforms/hiv/section-pmtct-history.xml
Comment (mogoodrich) [inline configuration/pih/htmlforms/hiv/section-pmtct-history.xml:8]: I suspect the formatter doesn't know how to format CSS and JavaScript found within <ifMode> tags since it's non-standard and specific to HTML Form Entry.  I've always been manually adding the tabs, which, admittedly, is a big pain.

@louidorjp  @brandones  thoughts on best practices here?

**Review by mogoodrich — APPROVED** [review]
@louidorjp  this  looks good to me.

It look like the was run through a formatter, which is good for consistency and something we should doing more of (thanks Louidor!) but I asked some questions on the PR regarding what standards we should use.  Interested in your thoughts and @brandones  as well.

(But I don't think that should block us from merging this PR in... do you have merge privileges to the ZL repo @louidorjp?)

File: configuration/pih/scripts/global/zl.js
Comment (louidorjp) [reply configuration/pih/scripts/global/zl.js:55]: The default for Dead is 5 and also 448 for Serological status.

File: configuration/pih/htmlforms/hiv/section-pmtct-history.xml
Comment (brandones) [reply configuration/pih/htmlforms/hiv/section-pmtct-history.xml:8]: Yeah I'm not inclined to worry too much about these. Manually maintaining the correct formatting is a pain, but is preferable to having it all unreadable.

File: configuration/pih/scripts/global/zl.js
Comment (brandones) [reply configuration/pih/scripts/global/zl.js:4]: OpenMRS 3.0 uses 2-space indents. This is pretty standard. See e.g. [Google JS Style Guide](https://google.github.io/styleguide/jsguide.html).

Let's just get this PR in and I'll add the editorconfig later.

Comment (louidorjp) [reply configuration/pih/scripts/global/zl.js:4]: @mogoodrich  I have'nt merge privileges to the ZL repo

**Comment by mogoodrich** [comment]
Thanks @louidorjp !  It looks like you (and Rikenson) do have merge privileges on the ZL repo, so we should figure out why you can't merge it in.

That being said, I'm happy to merge it in... but it's won't let me, because it looks like there are some conflicts on the PMTCT form between the changes you made and Ellen made, and it's not clear to me how to resolve them.  Can you take a look?  Thanks!

**Comment by brandones** [comment]
I think this is stale and was replaced by https://github.com/PIH/openmrs-config-zl/pull/99 ? Closing, feel free to re-open if I'm wrong about that.

**Pull request closed without merging**
